CMake: add option to compile with InterProcedural Optimizations (if available) (#1746)
ECP-WarpX/WarpX: Docs/source/building/cmake.rst

``WarpX_COMPUTE``             NOACC/**OMP**/CUDA/SYCL/HIP                  On-node, accelerated computing backend
``WarpX_DIMS``                **3**/2/RZ                                   Simulation dimensionality
``WarpX_EB``                  ON/**OFF**                                   Embedded boundary support

> ``WarpX_IPO                   ON/**OFF**                                   Compile WarpX with interprocedural optimization (will take more time)

Reviewer: Little formatting issue & simplification/shortening:
```suggestion
``WarpX_IPO``                 ON/**OFF**                                   Compile WarpX with interprocedural optimization (aka LTO)
```

> ``WarpX_IPO                   ON/**OFF**                                   Compile WarpX with interprocedural optimization (will take more time)

Reviewer: Little formatting issue & simplification/shortening:
```suggestion
``WarpX_IPO``                 ON/**OFF**                                   Compile WarpX with interprocedural optimization (aka LTO)
```

Author: Thanks for fixing the formatting issue! I also agree that `aka LTO` is more useful for a user than `(will take more time)`
